mps_youtube/main.py

```diff
@@ -3280,6 +3287,31 @@ def yt_url(url, print_title=0):
     if print_title:
         xprint(v.title)
 
+def yt_url_file(file_name):
+    """ Access a list of urls in a text file """
+
+    #Open and read the file
+    try:
+        fo = open(file_name, "r")
```

Comment:
It looks like `fo` is never explicitly closed. 
Perhaps try `with open(file_name, "r") as fo:`?